src/components/ui/weather-card/weather-Icon.ts

```diff
@@ -1,39 +1,5 @@
-import SunIcon from '../../../assets/icons/sun.png';
-import CloudIcon from '../../../assets/icons/cloud.png';
-import RainIcon from '../../../assets/icons/rain.png';
-import SnowIcon from '../../../assets/icons/snow.png';
-import ThunderstormIcon from '../../../assets/icons/thunderstorm.png';
-import { useState } from 'react';
-import { weather } from './const';
-
 const getIcons = () => {
```

Comment:
add to types.ts 
```
export interface WeaterhIconProps {
weathe: string;
}
```

`const WeatherIcon: FC<WeaterhIconProps> = ({weaather}) => {`